<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SampleApplyPatient</class>
 <widget class="QWidget" name="SampleApplyPatient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1216</width>
    <height>620</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1216</width>
    <height>620</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1216</width>
    <height>620</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>55</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>55</width>
         <height>22</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">width: 41px;
height: 22px;
font-family: PingFang SC;
font-weight: 500;
font-size: 22px;
color: #333333;
line-height: 35px;</string>
       </property>
       <property name="text">
        <string>姓名</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edit_name">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>215</width>
         <height>54</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>215</width>
         <height>54</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background: #F2F7FB;
border-radius: 25px;
border: 1px solid rgba(170,189,212,0.6);

font-family: PingFang SC;
font-weight: bold;
font-size: 24px;
color: #333333;
line-height: 35px;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>63</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>53</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>53</width>
         <height>22</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">width: 43px;
height: 22px;
font-family: PingFang SC;
font-weight: 500;
font-size: 22px;
color: #333333;
line-height: 35px;</string>
       </property>
       <property name="text">
        <string>年龄</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edit_age_val">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>56</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>56</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background: #F2F7FB;
border-radius: 28px;
border: 1px solid rgba(170,189,212,0.6);

font-family: PingFang SC;
font-weight: bold;
font-size: 24px;
color: #333333;
line-height: 35px;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_age_unit" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>200</horstretch>
         <verstretch>58</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>58</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>58</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>45</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>43</width>
         <height>22</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font-family: PingFang SC;
font-weight: 500;
font-size: 22px;
color: #333333;
line-height: 35px;</string>
       </property>
       <property name="text">
        <string>性别</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_sex" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>58</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>58</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>71</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>71</width>
         <height>22</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font-family: PingFang SC;
font-weight: 500;
font-size: 22px;
color: #333333;
line-height: 35px;</string>
       </property>
       <property name="text">
        <string>病人ID</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edit_patientID">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>230</width>
         <height>54</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>230</width>
         <height>54</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background: #F2F7FB;
border-radius: 25px;
border: 1px solid rgba(170,189,212,0.6);

font-family: PingFang SC;
font-weight: bold;
font-size: 24px;
color: #333333;
line-height: 35px;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>43</width>
         <height>22</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font-family: PingFang SC;
font-weight: 500;
font-size: 22px;
color: #333333;
line-height: 35px;</string>
       </property>
       <property name="text">
        <string>科室</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edit_department">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>240</width>
         <height>54</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>240</width>
         <height>54</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background: #F2F7FB;
border-radius: 27px;
border: 1px solid rgba(170,189,212,0.6);

font-family: PingFang SC;
font-weight: bold;
font-size: 24px;
color: #333333;
line-height: 35px;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>81</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>46</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>46</width>
         <height>21</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font-family: PingFang SC;
font-weight: 500;
font-size: 22px;
color: #333333;
line-height: 35px;</string>
       </property>
       <property name="text">
        <string>床号</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edit_bedID">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>245</width>
         <height>58</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>245</width>
         <height>58</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background: #F2F7FB;
border-radius: 29px;
border: 1px solid rgba(170,189,212,0.6);

font-family: PingFang SC;
font-weight: bold;
font-size: 24px;
color: #333333;
line-height: 35px;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>22</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font-family: PingFang SC;
font-weight: 500;
font-size: 22px;
color: #333333;
line-height: 35px;</string>
       </property>
       <property name="text">
        <string>送检者</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edit_inspection">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>230</width>
         <height>58</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>230</width>
         <height>58</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background: #F2F7FB;
border-radius: 29px;
border: 1px solid rgba(170,189,212,0.6);

font-family: PingFang SC;
font-weight: bold;
font-size: 24px;
color: #333333;
line-height: 35px;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>49</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>74</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>74</width>
         <height>22</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font-family: PingFang SC;
font-weight: 500;
font-size: 22px;
color: #333333;
line-height: 35px;</string>
       </property>
       <property name="text">
        <string>检验者</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edit_verifier">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>225</width>
         <height>58</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>225</width>
         <height>58</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background: #F2F7FB;
border-radius: 29px;
border: 1px solid rgba(170,189,212,0.6);

font-family: PingFang SC;
font-weight: bold;
font-size: 24px;
color: #333333;
line-height: 35px;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>58</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>67</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>67</width>
         <height>22</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font-family: PingFang SC;
font-weight: 500;
font-size: 22px;
color: #333333;
line-height: 35px;</string>
       </property>
       <property name="text">
        <string>审核者</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edit_auditor">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>245</width>
         <height>58</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>245</width>
         <height>58</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background: #F2F7FB;
border-radius: 29px;
border: 1px solid rgba(170,189,212,0.6);

font-family: PingFang SC;
font-weight: bold;
font-size: 24px;
color: #333333;
line-height: 35px;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>43</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>43</width>
         <height>22</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font-family: PingFang SC;
font-weight: 500;
font-size: 22px;
color: #333333;
line-height: 35px;</string>
       </property>
       <property name="text">
        <string>备注</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QTextEdit" name="edit_remark">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>1050</width>
         <height>400</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1050</width>
         <height>400</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background: #F2F7FB;
border-radius: 10px;
border: 1px solid rgba(170,189,212,0.6);
font-family: PingFang SC;
font-weight: 500;
font-size: 26px;
color: #333333;
line-height: 35px;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
